### PR TITLE
Remove "note that" in our documentation (as per style guide)

### PR DESCRIPTION
**Pull request by Hixie**
* Documentation improvements
* Remove "Note:" and "Note that" as per style guide

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM (at least tool code)

File: dev/conductor/core/lib/src/repository.dart [hunk 1]
```diff
@@ -129,8 +129,7 @@ abstract class Repository {
           'Fetch ${upstreamRemote.name} to ensure we have latest refs',
           workingDirectory: _checkoutDirectory!.path,
         );
-        // Note: if [initialRef] is a remote ref the checkout will be left in a
-        // detached HEAD state.
+        // If [initialRef] is a remote ref the checkout will be left in a detached HEAD state.
```
Comment (loic-sharma) [inline dev/conductor/core/lib/src/repository.dart:132]:
```suggestion
        // If [initialRef] is a remote ref, the checkout will be left in a detached HEAD state.
```

File: dev/tools/gen_keycodes/lib/physical_key_data.dart [hunk 2]
```diff
@@ -76,18 +76,21 @@ class PhysicalKeyData {
...
   /// # key 101 "KEY_LINEFEED"
   /// key 477   F12               FUNCTION
+  /// ```
   ///
   /// We parse the commented out lines as well as the non-commented lines, so
   /// that we can get names for all of the available scan codes, not just ones
   /// defined for the generic profile.
   ///
-  /// Also, note that some keys (notably MEDIA_EJECT) can be mapped to more than
+  /// Some keys (notably MEDIA_EJECT) can be mapped to more than
```
Comment (loic-sharma) [inline dev/tools/gen_keycodes/lib/physical_key_data.dart:93]:
```suggestion
  /// Some keys (notably `MEDIA_EJECT`) can be mapped to more than
```

File: packages/flutter/lib/src/material/dialog.dart [hunk 3]
```diff
@@ -546,21 +547,20 @@ class AlertDialog extends StatelessWidget {
...
-  /// The spacing between [actions] when the [OverflowBar] switches
-  /// to a column layout because the actions don't fit horizontally.
+  /// The spacing between [actions] when the [OverflowBar] switches to a column
+  /// layout because the actions don't fit horizontally.
   ///
   /// If the widgets in [actions] do not fit into a single row, they are
-  /// arranged into a column. This parameter provides additional
-  /// vertical space in between buttons when it does overflow.
+  /// arranged into a column. This parameter provides additional vertical space
+  /// in between buttons when it does overflow.
```
Comment (loic-sharma) [inline packages/flutter/lib/src/material/dialog.dart:555]:
```suggestion
  /// between buttons when it does overflow.
```

File: packages/flutter/lib/src/material/expansion_panel.dart [hunk 4]
```diff
@@ -139,19 +138,24 @@ class ExpansionPanelRadio extends ExpansionPanel {
...
 ///
-/// Note that [expansionCallback] behaves differently for [ExpansionPanelList]
-/// and [ExpansionPanelList.radio].
+/// The [expansionCallback] is called when the expansion state changes. For
+/// normal [ExpansionPanelList] widgets, it is the responsibilty of the parent
+/// widget to rebuild the [ExpansionPanelList] with updated values for
+/// [ExpansionPanel.isExpanded]. For [ExpansionPanelList.radio] widgets, the
+/// callback is invoked both for the previously open panel, which is closing,
+/// and the previously closed panel, which is opening, and the widget tracks the
+/// open state internally.
```
Comment (loic-sharma) [inline packages/flutter/lib/src/material/expansion_panel.dart:147]: The last "and the widget tracks the open state internally" statement feels disjointed to me. Consider reordering:

```suggestion
/// [ExpansionPanel.isExpanded]. For [ExpansionPanelList.radio] widgets, the
/// open state is tracked internally and the callback is invoked 
/// both for the previously open panel, which is closing, and the
// previously closed panel, which is opening.
```

(This suggestion's formatting is likely off)

File: packages/flutter/lib/src/material/expansion_tile.dart [hunk 5]
```diff
@@ -190,9 +190,9 @@ class ExpansionTile extends StatefulWidget {
   /// [children], and the `crossAxisAlignment` parameter is passed directly into the [Column].
   ///
   /// Modifying this property controls the cross axis alignment of each child
-  /// within its [Column]. Note that the width of the [Column] that houses
-  /// [children] will be the same as the widest child widget in [children]. It is
-  /// not necessarily the width of [Column] is equal to the width of expanded tile.
+  /// within its [Column]. The width of the [Column] that houses [children] will
+  /// be the same as the widest child widget in [children]. It is not
+  /// necessarily the width of [Column] is equal to the width of expanded tile.
```
Comment (loic-sharma) [inline packages/flutter/lib/src/material/expansion_tile.dart:195]: Consider something like:

```suggestion
  /// be the same as the widest child widget in [children]. The width of
  /// [Column] might not be equal to the width of the expanded tile.
```

File: packages/flutter/lib/src/material/text_theme.dart [hunk 6]
```diff
@@ -82,9 +82,9 @@ class TextTheme with Diagnosticable {
   /// If you do decide to create your own text theme, consider using one of
   /// those predefined themes as a starting point for [copyWith] or [apply].
   ///
-  /// Please note that you can not mix and match the 2018 styles with the 2021
-  /// styles. Only one or the other is allowed in this constructor. The 2018
-  /// styles will be deprecated and removed eventually.
+  /// The 2018 styles cannot be mixed with the 2021 styles. Only one or the
+  /// other is allowed in this constructor. The 2018 styles are deprecated and
+  /// will eventually be removed.
```
Comment (loic-sharma) [inline packages/flutter/lib/src/material/text_theme.dart:87]: The American school system is failing me here, but adverb should be after the verb right?

```suggestion
  /// will be removed eventually.
```

File: packages/flutter/lib/src/material/theme_data.dart [hunk 7]
```diff
@@ -1138,10 +1138,11 @@ class ThemeData with Diagnosticable {
   /// will aim to only support Material 3.
   ///
   /// ## Defaults
-  /// If a [ThemeData] is constructed with [useMaterial3] set to true, then
-  /// some properties will get updated defaults. Please note that
-  /// [ThemeData.copyWith] with [useMaterial3] set to true will
-  /// not change any of these properties in the resulting [ThemeData].
+  ///
+  /// If a [ThemeData] is _constructed_ with [useMaterial3] set to true, then
+  /// some properties will get updated defaults. The [ThemeData.copyWith] method
```
Comment (loic-sharma) [inline packages/flutter/lib/src/material/theme_data.dart:1143]: This is an unexpected call out, consider adding `However`:

```suggestion
  /// some properties will get updated defaults. However, the [ThemeData.copyWith] method
```

File: packages/flutter_tools/lib/src/commands/build_aar.dart [hunk 8]
```diff
@@ -99,9 +99,8 @@ class BuildAarCommand extends BuildSubCommand {
   final String description = 'Build a repository containing an AAR and a POM file.\n\n'
       'By default, AARs are built for `release`, `debug` and `profile`.\n'
       'The POM file is used to include the dependencies that the AAR was compiled against.\n'
-      'To learn more about how to use these artifacts, see '
-      'https://flutter.dev/go/build-aar\n'
-      'Note: this command builds applications assuming that the entrypoint is lib/main.dart. '
+      'To learn more about how to use these artifacts, see https://flutter.dev/go/build-aar\n'
+      'This command builds applications assuming that the entrypoint is lib/main.dart. '
       'This cannot currently be configured.';
```
Comment (loic-sharma) [inline packages/flutter_tools/lib/src/commands/build_aar.dart:104]:
```suggestion
      'This cannot be configured currently.';
```

File: packages/flutter_tools/lib/src/commands/build_aar.dart [hunk 9]
```diff
@@ -99,9 +99,8 @@ class BuildAarCommand extends BuildSubCommand {
   final String description = 'Build a repository containing an AAR and a POM file.\n\n'
       'By default, AARs are built for `release`, `debug` and `profile`.\n'
       'The POM file is used to include the dependencies that the AAR was compiled against.\n'
-      'To learn more about how to use these artifacts, see '
-      'https://flutter.dev/go/build-aar\n'
-      'Note: this command builds applications assuming that the entrypoint is lib/main.dart. '
+      'To learn more about how to use these artifacts, see https://flutter.dev/go/build-aar\n'
+      'This command builds applications assuming that the entrypoint is lib/main.dart. '
```
Comment (loic-sharma) [inline packages/flutter_tools/lib/src/commands/build_aar.dart:103]: Consider active voice here:

```suggestion
      'This command assumes that the entrypoint is lib/main.dart. '
```

I don't feel strongly though.

**Review by loic-sharma — APPROVED** [review]
Nice! 🎉

File: packages/flutter/lib/src/material/text_theme.dart (hunk 6 above)
Comment (Hixie) [reply packages/flutter/lib/src/material/text_theme.dart:87]: either is fine as far as i'm aware

File: packages/flutter_tools/lib/src/commands/build_aar.dart (hunk 8 above)
Comment (Hixie) [reply packages/flutter_tools/lib/src/commands/build_aar.dart:104]: This is one of those cases where the earlier adverb acts as emphasis, I think.

**Comment by Hixie** [comment]
Applied your fixes (and a couple others I noticed while doing so), modulo the adverb/verb ones per the comments above.

**Comment by Hixie** [comment]
thanks for the reviews, will land on green

File: packages/flutter/lib/src/material/text_theme.dart (hunk 6 above)
Comment (loic-sharma) [reply packages/flutter/lib/src/material/text_theme.dart:87]: Ah gotcha, feel free to ignore then :)

File: packages/flutter_tools/lib/src/commands/build_aar.dart (hunk 8 above)
Comment (loic-sharma) [reply packages/flutter_tools/lib/src/commands/build_aar.dart:104]: Feel free to ignore this, I don't feel strongly :)

**Pull request merged**